test/activation_based/test_nir_exchange.py

```
@@ -0,0 +1,293 @@
+
+
+def test_export_preserves_memories():
+    net = nn.Sequential(neuron.LIFNode(tau=2.0, v_reset=0.0))
+    x = torch.full((1, 2), 0.75)
+    net(x)
+    before = [value.clone() for value in base.memories(net)]
+
+    nir_exchange.export_to_nir(net, x)
+
+    after = list(base.memories(net))
+    assert all(torch.equal(x, y) for x, y in zip(before, after))
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)